docs/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 ## Catalog, Lambdas
 !-->
 ## Python API
+* [Added] Add docstring warnings for metadata ([#3156](https://github.com/quiltdata/quilt/pull/3156))
```

Comment:
I don't think it worth to add changelog entry for this. Also "added" is for more like for new features.

Reply:
Updated to [Changed]